content/docs/components-and-props.md

```diff
@@ -171,11 +171,11 @@ function Avatar(props) {
 }
 ```
 
-The `Avatar` doesn't need to know that it is being rendered inside a `Comment`. This is why we have given its prop a more generic name: `user` rather than `author`.
+`Avatar`-ը կարիք չունի իմանալու, որ արտապատկերվում է `Comment`-ի մեջ։ Դրա համար նրա prop-ին տվել ենք ավելի ընհանուր անուն\` `user`, փոխարենը `author`-ի։
 
-We recommend naming props from the component's own point of view rather than the context in which it is being used.
+Մենք խորհուրդ ենք տալիս անվանել prop-երը component-ի տեսանկյունից, ոչ թե կոնտեքստի, որում այն օգտագործվում է։
```

Comment:
Էս մասը մի քիչ սահուն չեմ կարողանում կարդալ, դրա համար առաջարկում եմ․ "ոչ թե կոնտեքստի, որում․․" -> "ոչ թե այն կոնտեքսի տեսանկյունից, որում․․․"

Reply:
This is why֊ը իմաստային ավելի լավ կլինի "Դրա համար" ֊> "Դա է պատճառը, որ"